### DISCUSSION TITLE
[CLI] Generate a mockoon environment from an OpenAPI YAML file without starting the server

### DISCUSSION THREAD

**Discussion by zerock54** [mockoon/mockoon#1128] [Help]
Hi,

I am using the CLI and I'd like to generate a Mockoon env (ideally, at a desired location) without starting the server.
Is that possible right now ?

Thanks

**Comment by 255kb** [comment]
Hi, you can already run a mock from an open API specification file, but it will not persist the Mockoon data file.
Though, this is currently being worked on. https://github.com/mockoon/mockoon/pull/772

**Reply by zerock54** [reply]
Thanks. This PR looks promising but I see that it hasn't been updated in over a year. 
I can continue working on it if that's possible. What do you think ?

**Reply by 255kb** [reply]
You can ask directly on the PR. 
One year is a long time. It's probably time to move on.

**Reply by 255kb** [reply]
@zerock54 I will ask the contributor and if there is no response after some days you can take over the work. I will merge on a separated branch from which you could continue.

**Reply by zerock54** [reply]
Ok perfect thanks 😊
